lib/generator._js

```diff
@@ -458,18 +454,17 @@ ScriptGenerator.prototype.generateDotNetDeploymentScript = function (templateFil
 ScriptGenerator.prototype.generateAspNetCoreScript = function (templateFileName, options, _) {
   argNotNull(templateFileName, 'templateFileName');
 
-  var fixedAspNetCoreProject = this.scriptType === ScriptType.batch ? fixPathSeperatorToWindows(options.aspNetCoreProject) : fixPathSeperatorToUnix(options.aspNetCoreProject);
-  var fixedRestore = this.scriptType === ScriptType.batch ? fixPathSeperatorToWindows(options.restore) : fixPathSeperatorToUnix(options.restore);
-  var fixedSolutionPath = this.scriptType === ScriptType.batch ? fixPathSeperatorToWindows(options.solutionPath) : fixPathSeperatorToUnix(options.solutionPath);
+  var fixedDotnetpublishArguments = this.scriptType === ScriptType.batch ? fixPathSeperatorToWindows(options.DotnetpublishArguments) : fixPathSeperatorToUnix(options.DotnetpublishArguments);
+  var fixedRestoreArguments = this.scriptType === ScriptType.batch ? fixPathSeperatorToWindows(options.RestoreArguments) : fixPathSeperatorToUnix(options.RestoreArguments);
```

Comment:
Yep, that should be fine; my JS is rough so I just hacked that in.

Let me know when these changes come up again; happy to review.